packages/react/src/components/Modal/Modal.mdx

```diff
@@ -34,6 +34,14 @@ Vi bruker native `autoFocus` på `Textfield` for å fokusere inputen i skjemaet.
 
 <Canvas of={ModalStories.ModalWithForm} />
 
+## Med egendefinert vidde
+
+Legg til `maxWdith`som `style` dersom du trenger egendefinert vidde på modalen.
```

Comment:
Bruk `max-width` For å sette egen max bredde på modalen. Default er `650px`.

Reply:
Blir ikke brukeren forvirret når vi sier de skal bruke max-width men så er det egentlig maxWidth de skal bruke?

Reply:
Dei kan enten bruke `maxWidth` i `style` proppen, eller `max-width` i ein css klasse. hm, @mimarz tanker?

Reply:
Vi bruker de faktisk css property navnene, dvs `max-width`. Proppen `style` til React er bare en abstraksjon de har laget for at du skal kunne sette styling med javascript.

Reply:
Allright, hadde nok satt pris på og fått vite at jeg da kunne bruke begge metodene for å sette det men hvis vi bare skal bruke css propertien så gjør jeg så